components/ai_chat/resources/page/stories/components_panel.tsx

```diff
@@ -34,6 +34,10 @@ import ErrorServiceOverloaded from '../components/alerts/error_service_overloade
 import LongConversationInfo from '../components/alerts/long_conversation_info'
 import WarningPremiumDisconnected from '../components/alerts/warning_premium_disconnected'
 import Attachments from '../components/attachments'
+import { createTextContentBlock } from '../../common/content_block'
+import ToolEvent from '../../untrusted_conversation_frame/components/assistant_response/tool_event'
+
+// TODO(petemill): Attempt to split this file up
```

Comment:
maybe file an issue for this instead of adding more TODOs for yourself :laughing:

Reply:
https://github.com/brave/brave-browser/issues/47810